src/core/tsi/s2a/handshaker/s2a_handshaker_client.h

```diff
@@ -46,6 +46,16 @@ typedef struct grpc_s2a_credentials_options grpc_s2a_credentials_options;
 typedef grpc_call_error (*s2a_grpc_caller)(grpc_call* call, const grpc_op* ops,
                                            size_t nops, grpc_closure* tag);
 
+/** The vtable for the S2A handshaker client operations. **/
+typedef struct s2a_handshaker_client_vtable {
```

Comment:
If we C++'fy `s2a_handshaker_client`, do we still need this v-table?

Reply:
We do not. This is removed in the next PR in this sequence (the implementation of the HS client), but I'll remove it here too.